ivy/functional/backends/paddle/layers.py

```diff
@@ -6,6 +6,7 @@
 import paddle
 import ivy
 from ivy.func_wrapper import with_unsupported_device_and_dtypes
+
```

Comment:
quite minor, but let's remove this blank line

Reply:
oh yes, done